kclvm/runtime/src/net/mod.rs

```diff
@@ -19,12 +19,78 @@ pub extern "C-unwind" fn kclvm_net_split_host_port(
     let kwargs = ptr_as_ref(kwargs);
     let ctx = mut_ptr_as_ref(ctx);
 
-    if let Some(string) = get_call_arg_str(args, kwargs, 0, Some("ip_end_point")) {
-        let mut list = ValueRef::list(None);
-        for s in string.split(':') {
-            list.list_append(&ValueRef::str(s));
+    if let Some(ip_end_point) = get_call_arg(args, kwargs, 0, Some("ip_end_point")) {
```

Comment:
can we use std::net?
https://doc.rust-lang.org/std/net/enum.SocketAddr.html

Reply:
`SocketAddr::parse_ascii` is listed as unstable. It also doesn't handle domain names, which this function needs to.